hail/python/hail/methods/qc.py

```diff
@@ -492,13 +514,363 @@ def n_discordant(counter):
+                          regions=vep_config.regions,
+                          requester_pays_project=requester_pays_project,
+                          env=get_env(-1, None, local_output_file),
+                          )
+
+        for f in hl.hadoop_ls(vep_input_path.name):
+            path = f['path']
+            part_name = os.path.basename(path)
+            if not part_name.startswith('part-'):
+                continue
+            part_id = int(part_name.split('-')[1])
+
+            local_input_file = '/io/input'
+            local_output_file = '/io/output.gz'
+
+            bb.create_job(vep_config.image,
+                          vep_config.command,
+                          attributes={'name': f'vep-{part_id}'},
+                          resources={'cpu': '1', 'memory': 'standard'},
+                          input_files=[(path, local_input_file)],
+                          output_files=[(local_output_file, f'{vep_output_path.name}/annotations/{part_name}.tsv.gz')],
+                          cloudfuse=[(vep_config.data_bucket, vep_config.data_mount, True)],
+                          regions=vep_config.regions,
+                          requester_pays_project=requester_pays_project,
+                          env=get_env(part_id, local_input_file, local_output_file),
+                          )
+
+    hl.export_vcf(ht, vep_input_path.name, parallel='header_per_shard')
+
+    timings = Timings()
+    name = 'vep(...)'
+
+    with timings.step("submit batch"):
+        bb = backend.bc.create_batch(token=token,
+                                     attributes={'name': backend.name_prefix + name, 'vep': '1', 'token': token},
+                                     cancel_after_n_failures=1)
+        build_vep_batch(bb)
+        b = bb.submit(disable_progress_bar=True)
+
+    with timings.step("wait batch"):
+        try:
+            status = b.wait(description=name,
+                            disable_progress_bar=backend.disable_progress_bar,
+                            progress=None)
+        except BaseException:
+            print('cancelling batch...')
+            b.cancel()
+            raise
+
+    with timings.step("parse status"):
+        if status['n_succeeded'] != status['n_jobs']:
+            failing_job = [job for job in b.jobs('!success')][0]
+            failing_job = b.get_job(failing_job['job_id'])
+            job_status = failing_job.status()
+            if 'status' in job_status:
+                if 'error' in job_status['status']:
+                    job_status['status']['error'] = yamlx.dump(job_status['status']['error'].strip())
+            logs = failing_job.log()
+            for k in logs:
+                logs[k] = yamlx.dump(logs[k].strip())
+            message = {'batch_status': status,
+                       'job_status': job_status,
+                       'log': logs}
+            log.error(yamlx.dump(message))
+            raise FatalError(message)
```

Comment:
I think we should not log and raise the same message. It will appear twice and overwhelm the user. Let's just raise.

I believe I've fixed the issues with needing to `.strip()` strings. I think this whole thing can now be:
```
message = {
    'batch_status': status,
    'job_status': failing_job.status(),
    'log': failing_job.log()
}
raise FatalError(yamlx.dump(message))
```